extension/js/common/api/account-servers/external-service.ts

```diff
@@ -171,18 +172,45 @@ export class ExternalService extends Api {
     } else if (method !== 'GET') {
       reqFmt = 'JSON';
     }
-    return await ExternalService.apiCall(
-      this.url,
-      path,
-      vals,
-      reqFmt,
-      progress,
-      {
-        ...headers,
-        ...(await this.authHdr()),
-      },
-      'json',
-      method
-    );
+    try {
+      return await ExternalService.apiCall(
+        this.url,
+        path,
+        vals,
+        reqFmt,
+        progress,
+        {
+          ...headers,
+          ...(await this.authHdr()),
+        },
+        'json',
+        method
+      );
+    } catch (firstAttemptErr) {
+      const idToken = await InMemoryStore.get(this.acctEmail, InMemoryStoreKeys.ID_TOKEN);
+      if (ApiErr.isAuthErr(firstAttemptErr) && idToken) {
+        // force refresh token
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
It seems `eslint-disable` not needed here

Reply:
Ok